src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -282,6 +281,7 @@ trait FastSync {
     }
 
     def assignWork(peer: ActorRef): Unit = {
+      //fastSyncStateStorage.putSyncState(initialSyncState.copy(bestBlockHeaderNumber = 3395264))
```

Comment:
(I hit the 'Approve' button hours ago.... but got your last just now .... you time travelling again? ;-) )